docs/admin/ambassador_pattern_linking.md

```diff
@@ -1,18 +1,17 @@
 <!--[metadata]>
 +++
+aliases = ["/articles/ambassador_pattern_linking/"]
```

Comment:
should the aliases start with `/engine/` ? Given that the current URL is https://docs.docker.com/engine/articles/ambassador_pattern_linking/